src/test/java/uk/gov/hmcts/reform/slc/services/UpdateLetterStatusJobTest.java

```diff
@@ -45,4 +55,44 @@ public void should_download_reports_if_ftp_is_available() {
 
         verify(ftpClient, times(1)).downloadReports();
     }
+
```

Comment:
How about the happy case to check if we delete the file when everything worked fine?

Reply:
added